BaragonAgentService/src/main/java/com/hubspot/baragon/agent/listeners/ResyncListener.java

```diff
@@ -61,12 +63,16 @@ public ResyncListener(LifecycleHelper lifecycleHelper,
 
   @Override
   public void stateChanged(CuratorFramework client, ConnectionState newState) {
-    if (newState == ConnectionState.RECONNECTED) {
+    if (newState.equals(ConnectionState.RECONNECTED)) {
```

Comment:
not a huge deal, but enums are pass-by-value so using `==` is okay (and some people prefer it since it isn't susceptible to NPEs)